### PR TITLE
chore(Version): Add bump versioning

### PR DESCRIPTION
**Pull request by ehdsouza**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,30 +2,40 @@ language: go
...
+
 go:
 - 1.10.x
+
 before_install:
 - openssl aes-256-cbc -K $encrypted_56d12e7424aa_key -iv $encrypted_56d12e7424aa_iv
   -in .env.enc -out .env -d
 - nvm install 8.3
 - npm install -g npm@6.x
+- sudo apt-get update
```
Comment (germanattanasio) [inline .travis.yml]: You add doing `apt-get update and installing python in every single build.

That seems a little bit too much.

How much time is this adding to the build?

File: .bumpversion.cfg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[bumpversion]
+current_version = 0.1.1
+commit = True
+tag = True
+
+[bumpversion:file:core/version.go]
+search = Version = '{current_version}'
```
Comment (germanattanasio) [inline .bumpversion.cfg]:
```
Version = "{current_version}"
```

File: .bumpversion.cfg [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+replace = Version = '{new_version}'
```
Comment (germanattanasio) [inline .bumpversion.cfg]: Version = "{new_version}"

File: .travis.yml (hunk 1 above)
Comment (ehdsouza) [reply .travis.yml]: Just 4 secs.
screen shot 2018-11-05 at 11 02 12 am

**Pull request merged**